cluster/ci/config/roles.yaml

```diff
@@ -39,40 +39,44 @@ items:
   - dustymabe
   - jlebon
 - kind: ClusterRoleBinding
-  apiVersion: v1
+  apiVersion: rbac.authorization.k8s.io/v1beta1
   metadata:
     name: ci-admins-sudoers
   subjects:
   - kind: Group
     name: ci-admins
   roleRef:
+    apiGroup: rbac.authorization.k8s.io
     name: sudoer
 - kind: ClusterRoleBinding
-  apiVersion: v1
+  apiVersion: rbac.authorization.k8s.io/v1beta1
   metadata:
     name: ci-admins-to-cluster-reader
   subjects:
   - kind: Group
     name: ci-admins
   roleRef:
+    apiGroup: rbac.authorization.k8s.io
     name: cluster-reader
 - kind: RoleBinding
-  apiVersion: v1
+  apiVersion: rbac.authorization.k8s.io/v1beta1
   metadata:
     name: authenticated-users-to-view
     namespace: ci
   subjects:
   - kind: SystemGroup
     name: system:authenticated
   roleRef:
+    apiGroup: rbac.authorization.k8s.io
     name: view
 - kind: RoleBinding
-  apiVersion: v1
+  apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Same last 2 changes since this is the same role binding in different namespaces.